tests/kernel/threads/thread_apis/src/main.c

```diff
@@ -471,6 +480,158 @@ void test_thread_join_deadlock(void)
 		      "couldn't join deadlock2_thread");
 }
 
+#define WAIT_TO_START_MS 100
+static void user_start_thread(void *p1, void *p2, void *p3)
+{
+	/* do nothing */
+}
```

Comment:
I guess there doxygen comments are missing here. Please add them. Also, please check other test cases in the file.

Reply:
@LeiW000  It seems like Xianglin has already updated it.